tapir/core/templatetags/core.py

```diff
@@ -105,6 +105,11 @@ def get_sidebar_link_groups(request):
             material_icon="priority_high",
             url=reverse_lazy("shifts:members_on_alert"),
         )
+        shifts_group.add_link(
+            display_name=_("Add a shift"),
```

Comment:
Depending whether it will be possible to update or delete shifts in the future, I would name it more generically. Such as "Edit shifts" or so